instructor/dsl/partial.py

```diff
@@ -191,10 +188,7 @@ async def extract_json_async(
                 if mode == Mode.ANTHROPIC_TOOLS:
                     yield chunk.delta.partial_json
                 elif chunk.choices:
-                    if mode == Mode.FUNCTIONS:
```

Comment:
lets not delete functions, since older openai openapi wrappers might use functions vs tools